pkgs/development/arduino/arduino-mk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchgit }:
+
+stdenv.mkDerivation rec {
+  version = "1.6.0";
+  name = "arduino-mk-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/sudar/Arduino-Makefile.git";
+    rev = "refs/tags/${version}";
+    sha256 = "0k646nm2n3kji4hp727467z0bicfa67w43ll06mxsdqssnbwgghi";
```

Comment:
Looks ilke the proper hash is 0flpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x, but also I recommend using fetchFromGitHub instead. Check this out for how to use it: https://nixos.org/nixpkgs/manual/#sec-sources